### PR TITLE
Doc clarifications

### PR DESCRIPTION
**Pull request by ewolinetz**
@bparees @soltysh PTAL

### CODE REVIEW

**Comment by ewolinetz** [comment]
cherry-picks from #212 included

File: docs/builder_image.md [hunk 1]
```diff
@@ -115,7 +116,7 @@ The `run` script is responsible for executing your application.
 
 ## save-artifacts
 
-The `save-artifacts` script is responsible for gathering all the dependencies into a tar file and streaming it to the standard output.  The existance of this can speed up the following build processes (eg. for Ruby - gems installed by Bundler, for Java - `.m2` contents, etc.).
+The `save-artifacts` script is responsible for gathering all the dependencies into a tar file and streaming it to the standard output (eg. for Ruby - gems installed by Bundler, for Java - `.m2` contents, etc.).  The existance of this can speed up the following build processes.
```
Comment (bparees) [inline docs/builder_image.md]: existence.

**Comment by bparees** [comment]
one nit and lgtm.  
and thanks @ianmiell for the original commit.

**Comment by ewolinetz** [comment]
nit addressed

**Comment by bparees** [comment]
[merge]

**Pull request merged**